.changeset/cold-news-glow.md

```diff
@@ -0,0 +1,14 @@
+---
+"@localyze-pluto/components": major
+---
+
+[ContentCard]
+
+> Breaking change: `iconUrl` prop is now `icon` and accepts a ReactNode
+
+Minor changes:
+
+- Set Icon wrapper border radius to 8px (shows now like a square)
+- Added new `topRightElement` prop to allow add element in the top right corner of the card
+- Added new `lowOpacity` prop to set the opacity of the card content without include the `topRightElement`
```

Comment:
Not setting the opacity on the topRightElement feels super specific, like something that should not be part of the DS, especially considering that the topRightElement could be anything... like, why is it only the topRIghtElement and not the icon too...

I don't have an alternative, I'm just thinking out loud here

Reply:
Yeah, I get it. This was meant for our scenario on the partners page. We want to display the card with low opacity but still show the badge. 🤷🏻‍♀️

Reply:
Yeah...